examples/taskworkflow.go

```diff
@@ -115,11 +122,11 @@ func taskWorkflow(containerCIDR string, gateway string) (err error) {
 		}
 	}()
 
-	log.Printf("Successfully pulled %s image\n", image.Name())
+	log.Printf("Successfully pulled %s image from %s\n", image.Name(), snapshotter)
```

Comment:
s/from/with/

Reply:
Thanks. Fixed on https://github.com/firecracker-microvm/firecracker-containerd/pull/300/commits/cf050b10ec05105122509892206000107247d0fe and since the commit is the first commit in this PR, I have to force-push...